packages/generator/src/generator.js

```diff
@@ -37,7 +37,7 @@ export default class Generator {
 
     // Payloads for full static
     if (this.isFullStatic) {
-      consola.info('Full static mode activated')
+      consola.info('Generating with full static mode')
```

Comment:
Could this whole if be moved into a separate function (including the if)? For nuxt-generate-cluster Im not calling this method but instead calli each function that this method calls either from the master or worker thread. As this if is not abstracted we'd need to copy this implementation ourselves, see eg this pr: https://github.com/nuxt-community/nuxt-generate-cluster/pull/32

Thanks :)

Reply:
@pimlie I'm not sure related to this PR but would you mind directly committing your idea on this branch or new PR?

Reply:
@pi0 Done, it should be fine to just move this if into `initiate`. We call this method in nuxt-generate-cluster already and it would need to be set for both master as workers.

Reply:
Thanks, I see :) Is it also ok with  d8d6045 and 51d693d?

Reply:
Yep, should be ok too. Thanks :)